contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
I have to insist, don't take it personally :)

Being very big hasn't been a problem. Being impossible to find or know what's tested and what is not *has* been a problem, we used to have "theme-based" tests and it created a huge mess where people were reimplementing tests twice that did the same thing in two different places.